src/components/subscriptions/LicenseStatus.jsx

```diff
@@ -15,7 +15,6 @@ export default function LicenseStatus({ user }) {
           };
         case 'assigned':
           return {
-            iconClassName: 'fa-hourglass-half text-muted',
```

Comment:
Why are we removing the hourglass icon here?

Reply:
Fixed.